data/building-hours/1-2-pause-kitchen.yaml

```diff
@@ -6,9 +6,10 @@ schedule:
   - title: Daylight
     notes: The late night menu starts at 9 p.m.
     hours:
-      - {days: [Mo, Tu, We, Th], from: '10:30am', to: '12:00am'}
-      - {days: [Fr, Sa], from: '10:30am', to: '9:00pm'}
-      - {days: [Su], from: '10:30am', to: '12:00am'}
+      - {days: [Mo, We], from: '10:30am', to: '12:00am'}
```

Comment:
Do Monday/Wednesday not have a Late Night menu? Or should they also "close" at 9pm with a Late Night section from 9-midnight?